www/include/configuration/configObject/host/listHost.ihtml

```diff
@@ -67,7 +67,7 @@
 			<a href="{$elemArr[elem].RowMenu_link}"><img src="{$elemArr[elem].RowMenu_icone}" class="ico-18  margin_right" />{$elemArr[elem].RowMenu_name}</a>
 		</td>
 		<td class="ListColCenter">
-            <a href='./main.php?p=602&search_h={$elemArr[elem].RowMenu_name}&search_s='>
+            <a href='./main.php?p=602&search=&searchH={$elemArr[elem].RowMenu_name}&searchS='>
```

Comment:
please remove "&search" as it's not a filter, nor used in this page

Reply:
for this task you only need to modify the line 70 as this :
`<a href='./main.php?p=602&searchH={$elemArr[elem].RowMenu_name}'>`